test/verify/check-pages

```diff
@@ -563,7 +564,8 @@ OnCalendar=daily
 
         stuff = os.path.join(self.vm_tmpdir, "stuff")
         # prepare a directory for testing file autocomplete widget
-        m.execute("mkdir -p {0}/dir1; touch {0}/file1.txt".format(stuff))
+        m.execute(f"mkdir -p {stuff}/dir1")
```

Comment:
Confusing, but correct.

Could also be re-written as `mkdir` (without `-p`) *after* the `write()`.

Reply:
It was indeed confusing before, as I initially mis-read it as "make the dir for the touched file", but that's not what it did. It's at least less confusing now.

The `-p` is indeed unnecessary, but TBH I don't feel like running this again just for this.